matRad_generateStfSinglePencilBeam.m

```diff
@@ -0,0 +1,86 @@
+function stf = matRad_generateStfSinglePencilBeam(ct,cst,pln,SAD)
```

Comment:
see above comment regarding the SAD argument.

Reply:
Done.